packages/esm-patient-medications-app/src/add-drug-order/drug-search/drug-search.resource.tsx

```diff
@@ -89,83 +89,83 @@ export function getTemplateOrderBasketItem(
   return template
     ? {
         action: 'NEW',
-        display: drug.display,
-        drug,
-        unit:
-          getDefault(template.template, 'unit') ?? drug?.dosageForm
-            ? {
-                value: drug?.dosageForm?.display,
-                valueCoded: drug?.dosageForm?.uuid,
-              }
-            : null,
-        dosage: getDefault(template.template, 'dose')?.value,
-        frequency: getDefault(template.template, 'frequency'),
-        route: getDefault(template.template, 'route'),
-        commonMedicationName: drug.display,
-        isFreeTextDosage: false,
-        patientInstructions: '',
+        dateActivated: new Date(),
```

Comment:
This is the key change: renaming this property from `startDate` to `dateActivated`.